frontend/src/tournaments/roundRobin/robinCrosstablePage.tsx

```diff
@@ -0,0 +1,143 @@
+import CohortIcon from '@/scoreboard/CohortIcon';
+import {
+    Box,
+    Card,
+    Container,
+    FormControl,
+    InputLabel,
+    MenuItem,
+    Select,
+    SelectChangeEvent,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow,
+    Typography,
+} from '@mui/material';
+import React, { useEffect, useState } from 'react';
+import {
+    cohorts,
+    fetchTournamentData,
+    fetchTournamentIds,
+    TournamentData,
+} from './roundRobinApi';
+
+const Crosstable: React.FC = () => {
+    const [selectedCohort, setSelectedCohort] = useState<number>(0);
+    const [tournamentIds, setTournamentIds] = useState<string[]>([]);
+    const [tournamentData, setTournamentData] = useState<TournamentData[]>([]);
+
+    const handleCohortChange = (event: SelectChangeEvent<number>) => {
+        setSelectedCohort(Number(event.target.value));
+    };
+
+    useEffect(() => {
+        if (selectedCohort !== 0) {
+            fetchTournamentIds(selectedCohort)
+                .then(setTournamentIds)
+                .catch(console.error);
+        }
+    }, [selectedCohort]);
+
+    useEffect(() => {
+        if (tournamentIds.length > 0) {
+            setTournamentData([]); // Clear previous tournament data
+            Promise.all(tournamentIds.map((id) => fetchTournamentData(id)))
+                .then(setTournamentData)
+                .catch(console.error);
+        }
+    }, [tournamentIds]);
+
+    useEffect(() => {
+        if (selectedCohort !== 0) {
+            fetchTournamentIds(selectedCohort);
+        }
+    }, [selectedCohort]);
+
+    useEffect(() => {
```

Comment:
This seems to be a duplicate of the second `useEffect` and seems like it can be deleted?

Reply:
yes, typo